README.md

```diff
@@ -67,18 +64,41 @@ tfenv use
 # Go to stack which you want to deploy
 cd environments/prod
 
-# Verify files
+# Initialize a working directory and install dependencies (terraform modules)
+terraform init
+
+# Verify files (run linters)
 pre-commit run --all-files
 
-# Verify changes
-terraform init
+# Verify changes which will be applied on AWS
 terraform plan
 
-# Deploy changes
-# Do it only when you are sure of what you are doing!
+# Deploy changes: Do it only when you are sure of what you are doing!
 terraform apply
 ```
 
+To simplify work you can use command defined in `Makefile`:
```

Comment:
Makefile reminds me about computer science C lectures 😅

Reply:
`Make` is very simple and help me a lot (I don't have to jump between directories).

Do you have any other idea how to simplify that? What else could be used?

I've chosen Make because is installed on MACs and very, very simple ... good enough for this work :)